wallaby.js

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require("dotenv").config();
+
+module.exports = function (_wallaby) {
+  const packageNames = ["backend", "encoding", "crypto", "frontend"];
+  const packages =
+    packageNames.length === 1 ? packageNames[0] : `{${packageNames.join(",")}}`;
```

Comment:
```suggestion
    packageNames.join(",");
```

Reply:
Your suggestion drops `{` and `}`.

If there is more than one directory name in the array we want `"{one,two}"`, otherwise we just want `"one", because `"{one}"` is illegal.

Reply:
Oh! I didn't notice the brackets. Then the code is correct